Gruntfile.js

```diff
@@ -59,6 +59,44 @@ module.exports = function(grunt) {
           {expand: true, cwd: 'test/fixtures/', src: ['**'], dest: './'}
         ]
       },
+      zipCreateEmptyArchiveTrue: {
+        options: {
+          createEmptyArchive: true,
+          archive: function () {
+            return 'tmp/zip_must_be_created_1.zip';
+          }
+        },
+        files: [{
+          expand: true,
+          cwd: 'test/fixtures/',
+          src: ['NotExistingFilePath.js']
+        }]
+      },
+      zipCreateEmptyArchiveFalse: {
+        options: {
+          createEmptyArchive: false,
+          archive: function () {
+            return 'tmp/zip_should_not_be_created.zip';
+          }
+        },
+        files: [{
+          expand: true,
+          cwd: 'test/fixtures/',
+          src: ['NotExistingFilePath.js']
+        }]
+      },
+      zipNoCreateEmptyArchiveOption: {
+        options: {
+          archive: function () {
+            return 'tmp/zip_must_be_created_1.zip';
```

Comment:
Oh. My bad. There must be two names for 2 cases. I will explain why.

There are 3 tests. 1nd and 3rd tests expect archive to be created (option is true or no option). If I will make same names it is possible that 1st test went good, 3rd test went bad (archive is not created) and both expects will be good, but it is not true. I think I must use 2 different names there. What do you think?